concepts/observability-strategy/configurable-logging/01-logging-groundwork.md

```diff
@@ -1,10 +1,12 @@
-# Groundwork
+# Configurable Logging: Groundwork
 
 ## Current Situation and Motivation
 
-![a](./assets/current.drawio.svg)
+![a](./assets/logging-current.drawio.svg)
 
-In the diagram, you see that all three observability aspects (log, trace, metric) provide a preconfigured backend with visualisations. However, they don't provide a neutral and unified way to integrate backends outside of the cluster. The tracing stack provides no way to centrally push trace data to the outside. Logging can be configured much more flexibly and neutrally, however, the configuration must be done during installation so it isn't lost at the next Kyma upgrade process. Furthermore, it is not easy to use to mix and match different integrations, because you need to deal with one centralized configuration (the fluent-bit config).
+In the current setup, all three observability aspects (log, trace, metric) provide a preconfigured backend with visualisations. However, they don't provide a neutral and unified way to integrate backends outside of the cluster.
```

Comment:
Why do we talk about all three observability aspects (and in line 9, about tracing specifically) in the logging-groundwork document?
The monitoring-groundwork doc focuses on monitoring only.